src/main/services/i18n/locales/README.md

```diff
@@ -3,27 +3,16 @@
 ## Add new Locales
   - Make a duplicate of the `en` folder, then rename it to the necessary [two-letter locale code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).
   - Translate each value in each of the files.
-  - Add new [prop](https://github.com/massCodeIO/massCode/blob/master/src/main/services/i18n/index.ts#L26) to i18n where the prop name corresponds to the name of the folder
+  - Add new [prop](https://github.com/massCodeIO/massCode/blob/master/src/main/services/i18n/language.ts#L1) to i18n where the prop name corresponds to the name of the folder
 ```js
 ...
   // 'en' for prop & 'English' for value
-  i18next.addResourceBundle(lng, 'language', {
+  export const language = {
     ...
     en: 'English',
     ...
-  })
-...
-```
-  - Add new [option](https://github.com/massCodeIO/massCode/blob/master/src/renderer/components/preferences/LanguagePreferences.vue#L42) for language selector where the name of the language corresponds to the name of the folder
-
-```js
-...
-  // 'en' for label & value
-  {
-    label: i18n.t('preferences:language.en'),
-    value: 'en'
   }
 ...
 ```
-
+  - commit your changes, remarks like this: `feat(i18n): add Chinese traditional locale`
```

Comment:
Of course, testing should be followed, but if some minor modifications are made through github, the restrictions of the rules will not take effect. Would it be better to be explicit here?

Reply:
In general the name of the commit does not matter, because it will be a squash merge. I understand why you wrote that, I wrote about it to the person who created the PR (Portuguese) . But it was as an addition to the fact that he is not attentive )

Reply:
Hmm, I understand.